android/first-ddc-app/scheduler/src/main/java/my/first/ddc/app/MyActivity.java

```diff
@@ -64,14 +63,10 @@ private void cancel() {
     @SuppressLint("MissingPermission")
     private SchedulerSettingsBuilder requestBuilder() {
         if (PermissionUtils.allGranted(this,
-                Manifest.permission.READ_PHONE_STATE,
                 Manifest.permission.GET_ACCOUNTS)) {
-            final TelephonyManager telephonyManager = (TelephonyManager)
-                    getSystemService(TELEPHONY_SERVICE);
-            if (telephonyManager != null) {
-                return ScheduledDDCFactory.setup(this, SYSTEM_ID, telephonyManager.getDeviceId(), DeviceIdType.IMEI)
+           return ScheduledDDCFactory.setup(this, SYSTEM_ID, "fakeIMEI", DeviceIdType.IMEI)
```

Comment:
are we sure we want the public test-app not to use the IMEI? I am of two minds on this.